src/openforms/emails/api/serializers.py

```diff
@@ -15,3 +18,23 @@ class Meta:
             "content",
             "translations",
         )
+
+    def to_internal_value(self, data):
+        if content := data.get("content"):
+            data["content"] = self._add_cosign_templatetag(content)
+
+        return super().to_internal_value(data)
+
+    def _add_cosign_templatetag(self, content):
+        is_import = self.context.get("is_import", False)
+
+        if not is_import:
+            return content
+
+        pattern = re.compile(r"\{%\s?cosign_information\s?%\}")
```

Comment:
you typically want to put that at the module level so that the regex is compiled once and then the cache is used, now it's being compiled every time this method runs so you might as well just use `re.search(r"\{%\s?cosign_information\s?%\}", content)`